tests/__init__.py

```
@@ -216,9 +281,9 @@
 		doc_suite.addTests(doctest.DocTestSuite(module=module, test_finder=finder))
 	except ValueError as e:
 		# ValueError is raised when no tests are found
-		print(f"No doctests found in {module.__name__}: {e}")
+		_LOGGER.warning(f"No doctests found in {module.__name__}: {e}")
 	except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)